lib/setupmanagers/qemuhck/qemu_machine.rb

```diff
@@ -303,13 +316,10 @@ def process_devices
         load_device_info(device_info)
       end
 
-      unless device_infos.any? { |d| d['type'] == 'vga' }
-        vga_info = read_device(@config['platforms_defaults']['vga_device'])
-        device_infos << vga_info
-        load_device_info(vga_info)
-      end
+      device_infos = add_missing_default_devices(device_infos)
```

Comment:
I prefer:
```suggestion
      add_missing_default_devices(device_infos)
```

Having an assignment for `device_infos` gives a false idea that the returned `Hash` is different from the original `Hash`, but they are actually the same.

Reply:
Fixed